### ISSUE TITLE
files:ignore are not ignored

### ISSUE THREAD

**Issue by sergejmueller** [skovhus/SublimeLinter-contrib-sass-lint#14]
Files that I have declared in _.sass-lint.yml_ as _ignore_ will **not** be ignored. Terminals _sass-lint_ works fine and all _files:ignore_ files are skipped.

**Comment by skovhus** [comment]
Hm, interesting. Which version of `sass-lint` are you using? And do you use the same version when running in the terminal AND in Sublime? So if you have `sass-lint` installed as a local node modules inside the project, Sublime will use that.

**Comment by adrianosmond** [comment]
I have the same issue.

I'm running v1.9.1 in both the terminal and node_modules.

**Comment by skovhus** [comment]
Any suggestions or pull requests with a fix are more than welcome. 😊

**Comment by WillsB3** [comment]
I'm also having this problem with 1.0.4 and sass-lint 1.10.2. Running `sass-lint -v -q` from the command line correctly ignores the files in my `.sass-lint.yml` config file's `files` section which looks like this:

```yaml
files:
  include: 'source/sass/**/*.s+(a|c)ss'
  ignore:
    - 'source/sass/legacy/**/*.*'
    - 'source/sass/lib/**/*.*'
```

However Sublime is reporting linting warnings for these files.

**Comment by dennis-best** [comment]
When I ran my sass-lint.yml file through a validator (http://www.yamllint.com/) it said this was bad:
```
files:
  include: 'src/sass/**/*.scss'
  ignore:
    - 'src/sass/_shame.scss'
    - 'src/sass/test/_tests.scss'
    - 'src/sass/vendor/**/*.*'
```
This, on the other hand, is valid:
```
files:
  include:
    - 'src/sass/**/*.scss'
  ignore:
    - 'src/sass/_shame.scss'
    - 'src/sass/test/_tests.scss'
    - 'src/sass/vendor/**/*.*'
```
Since, I can't replicate this error myself, I'll leave this here and hope it helps someone.
